File: assignment_1/Store.java
// This is the Store class - stores info about the main electronic store
public class Store {
    int storeId;        
    String storeName;   
    String location;    
    String contact;     

    // This function displays all store details
    // No parameters, returns nothing
    void displayStore() {
        System.out.println("Store ID : " + storeId);
        System.out.println("Name     : " + storeName);
        System.out.println("Location : " + location);
        System.out.println("Contact  : " + contact);
    }
}

File: python/store.py
class Store:
    def __init__(self):
        self.store_id = 0
        self.name = ""
        self.Location = ""
        self.Contact = ""

    def displayStore(self):
        print("Store ID :", self.store_id)
        print("Name     :", self.name)
        print("Location :", self.Location)
        print("Contact  :", self.Contact)